Logger/FileLogger.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Logger
+{
+    using System;
+    using System.IO;
+
+    public class FileLogger : BaseLogger
+    {
+        public FileLogger(string? filePath)
+        {
+            this.FilePath = filePath ?? throw new ArgumentNullException($"FilePath was null in the {nameof(FileLogger)}");
```

Comment:
```suggestion
            FilePath = filePath ?? throw new ArgumentNullException($"FilePath was null in the {nameof(FileLogger)}");
```
No need to use the `this` keyword. C# is case sensitive.